include/llbuild/Basic/RedirectedProcess.h

```diff
@@ -0,0 +1,42 @@
+//===- RedirectedProcess.h --------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLBUILD_BASIC_REDIRECTEDPROCESS_H
+#define LLBUILD_BASIC_REDIRECTEDPROCESS_H
+
+#include <signal.h>
+
+namespace llbuild {
+namespace basic {
+class RedirectedProcess {
+ public:
+  using ProcessId = pid_t;
+  ProcessId processId;
+
+  RedirectedProcess(ProcessId processId) { this->processId = processId; }
+
+  void kill(int signal);
```

Comment:
I should add some doc comments, I'll do that after you review the PR.
Also, from #98:

> What are the return values of things like kill() presumably false on failure, but always good to call out? I would also call kill something more English like sendSignal.

We never use the return values so I went for a void method. I should also rename this to `sendSignal` after you take a look